testing/mod.ts

```diff
@@ -124,8 +124,8 @@ function previousPrinted(name: string, results: TestResults): boolean {
 async function createTestCase(
   stats: TestStats,
   results: TestResults,
-  { fn, name }: TestDefinition,
-  exitOnFail: boolean
+  exitOnFail: boolean,
+  { fn, name }: TestDefinition
```

Comment:
Why didn't any tests fail before? Is there some API that's not being exercised?

cc @chiefbiiko

Reply:
As it's not exported, the only way to test it would be to do an assert in `runTests()` for `runTests()`. Maybe chiefbiiko has a clue to test it?

Reply:
Something isn't being exercised - otherwise #307 wouldn't have been green. We should add a test with this commit that was failing before.
Maybe @chiefbiiko can help.

Reply:
@ry @zekth Yea, sorry, I missed adding a test for parallel execution. See patch below.